cdap-data-fabric-tests/src/test/java/co/cask/cdap/data/stream/DFSStreamFileJanitorTest.java

```diff
@@ -110,6 +115,7 @@ protected void configure() {
         @Override
         protected void configure() {
           bind(MetadataStore.class).to(NoOpMetadataStore.class);
+          bind(OwnerStore.class).to(InMemoryOwnerStore.class);
```

Comment:
I think you need singleton for the `InMemoryOwnerStore` as you have everything as an in-memory map.